core/tests/protractor_utils/workflow.js

```diff
@@ -58,19 +58,13 @@ var createAndPublishExploration = function(
     title, category, objective, language) {
   createExploration();
   editor.setContent(forms.toRichText('new exploration'));
-  editor.setInteraction('TextInput');
-  editor.setDefaultOutcome(null, 'final state', true);
+  editor.setInteraction('EndExploration');
```

Comment:
@seanlip

EDIT:

The changes break the existing tests. I'll fix those issues.

Sorry for the confusion!